appengine/building-an-app/build/server.js

```diff
@@ -14,7 +14,7 @@
 
 'use strict';
 
-// [START app]
```

Comment:
Change affects one region tag that is still active. https://github.com/GoogleCloudPlatform/nodejs-docs-samples/pull/3617/checks?check_run_id=21334110961